app/modules/users/permissions/rules.py

```diff
@@ -149,36 +148,16 @@
     OBJECT_USER_MAP = {
         ('SiteSetting', AccessOperation.WRITE): ['is_admin'],
         ('SiteSetting', AccessOperation.DELETE): ['is_admin'],
-        ('Asset', AccessOperation.READ): [
-            'is_admin',
-            'is_data_manager',
-            'is_researcher',
-        ],
+        ('Asset', AccessOperation.READ): ['is_admin', 'is_researcher'],
         ('AssetGroup', AccessOperation.READ_PRIVILEGED): ['is_staff'],
-        ('AssetGroupSighting', AccessOperation.READ): [
-            'is_admin',
-            'is_data_manager',
-        ],
-        ('AssetGroupSighting', AccessOperation.WRITE): [
-            'is_admin',
-            'is_data_manager',
-        ],
-        ('AssetGroupSighting', AccessOperation.DELETE): [
-            'is_admin',
-            'is_data_manager',
-        ],
-        ('Individual', AccessOperation.READ): [
-            'is_staff',
-            'is_data_manager',
-            'is_researcher',
-        ],
-        ('Individual', AccessOperation.WRITE): ['is_data_manager'],
+        ('AssetGroupSighting', AccessOperation.READ): ['is_admin'],
+        ('AssetGroupSighting', AccessOperation.WRITE): ['is_admin'],
+        ('AssetGroupSighting', AccessOperation.DELETE): ['is_admin'],
+        ('Individual', AccessOperation.READ): ['is_admin', 'is_researcher'],
```

Comment:
it looks like this one got changed from `is_staff` to `is_admin`.  is this intentional (like it was a bug it was staff before)?
kinda guessing it might have been cuz that `is_staff` is like the oddball here.

Reply:
correct, staff is handled by a separate catchall outside of this